.gitignore

```diff
@@ -15,3 +15,4 @@
 /coverage
 /node_modules
 /npm-debug.log
+.DS_Store
```

Comment:
It looks good, except for this, which I think should be in a local gitignore file :-)